src/components/v5/common/ActionsContent/partials/UserSelect/hooks.ts

```diff
@@ -26,3 +31,30 @@ export const useUserSelect = (): UserSelectHookProps => {
     isAccordion: false,
   };
 };
+
+export const useVerifyRecipient = () => {
```

Comment:
Maybe it will be better to add isVerified prop to user object in `useUserSelect` hook? I think this will reduce the amount of logic in other components